tests/pact/yield.repl

```diff
@@ -4,8 +4,9 @@
 
 (module yieldtest 'k
```

Comment:
Need coverage of runtime rollback rejection for cross-chain.

Reply:
Added to `yield-rollback.repl`, which also covers a mix of steps with rollbacks, yields, and resumes.